console-frontend/src/components/hydra-console/HydraConsole.js

```diff
@@ -0,0 +1,613 @@
+import React, { Component } from "react";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import Input from "@material-ui/core/Input";
+import Grid from "@material-ui/core/Grid";
+import GuiTheme from "../../app/gui-theme";
+import { withStyles } from "@material-ui/styles";
+
+// Custom imports
+import { Scrollbars } from "react-custom-scrollbars";
+
+// Local components
+import EndpointsButtons from "./endpoints-buttons/EndpointsButtons";
+import OperationsButtons from "./operations-buttons/OperationsButtons";
+import PropertiesEditor from "./properties-editor/PropertiesEditor";
+import PrintObject from "./print-components/PrintObject";
+import PrintArray from "./print-components/PrintArray";
+// utils imports
+import {
+  isArray,
+  isObject,
+  setInLocalStorage,
+  getFromLocalStorage,
+  jsonStringifyReplacer,
+} from "../../utils/utils";
```

Comment:
Can we make this import better somehow? Maybe improve the directory structure a bit?

Reply:
I think we can make it little more declarative by putting utils.js in lib folder, or another thing we can do is, in utils folder we can separate files like local-storage-helpers.js (other files similar to this) containing utility functions like getting from local storage and setting values in local storage. Which approach do you think we should move ahead with?